.github/workflows/ci.yaml

```diff
@@ -0,0 +1,84 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+#
+# See https://github.com/r-lib/actions/tree/master/examples#readme for
+# additional example workflows available for the R community.
+
+name: BuildAndTest
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+env:
+  SERVER: ${{ secrets.SERVER }}
+  USER: ${{ secrets.USER }}
+  PASSWORD: ${{ secrets.PASSWORD }}
+  PASSWORD_AIRLINE_USER: ${{ secrets.PASSWORD_AIRLINE_USER }}
+  PASSWORD_AIRLINE_USER_DBOWNER: ${{ secrets.PASSWORD_AIRLINE_USER_DBOWNER }}
+  DATABASE: ${{ secrets.DATABASE }}
+  DRIVER: "{ODBC Driver 17 for SQL Server}"
+
+jobs:
+  R:
+    runs-on: windows-latest
+    strategy:
+      matrix:
+        r-version: [3.5]
+
+    env:
+      # Define CI to skip some test case.
+      CI: True
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up R ${{ matrix.r-version }}
+        uses: r-lib/actions/setup-r@v1
+        with:
+          r-version: ${{ matrix.r-version }}
+      - name: Install dependencies
+        working-directory: ./R
+        run: |
+          install.packages(c("remotes", "rcmdcheck"))
+          remotes::install_deps(dependencies = TRUE)
+        shell: Rscript {0}
+      - name: Check
+        working-directory: ./R
+        run: rcmdcheck::rcmdcheck(args = "--no-manual", error_on = "error")
+        shell: Rscript {0}
+
+  Python:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [3.7]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install odbc driver
+      working-directory: ./Python
+      run: |
+        sudo apt-get update
+        sudo ACCEPT_EULA=Y apt-get install -y msodbcsql17
+    - name: Install dependencies
+      working-directory: ./Python
+      run: |
+        python -m pip install --upgrade pip
+        python -m pip install flake8 pytest
+        if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
+    - name: Run build script
+      working-directory: ./Python
+      run: ./buildandinstall.sh
```

Comment:
why is this script used if its empty now ? I see that all lines have been removed from that script...